src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
+    },
+    multiply: function (num) {
+      return this.result * num;
```

Comment:
For the `multiply` method, use the `*=` operator to multiply the existing value.

BAD EXAMPLE:
 return this.result * num;

GOOD EXAMPLE:
 this.result *= num;

Please update the `multiply` method accordingly.